packages/node_modules/@webex/widget-speed-dial/src/SpeedDialModal.tsx

```diff
@@ -40,18 +40,19 @@ export const SpeedDialModal = ({
   const { t } = useTranslation('WebexSpeedDials');
 
   return (
+    <div aria-modal="true" role="dialog" aria-label={t('webex.calling') + "," + headerText + "," + t('voiceover.addSpeedDial.windowDialog')}>
     <GenericModal isOpen={isOpen} className={cssClasses}>
       <Flex className={sc('header')} direction="column">
         <Flex className={sc('header-text')}>
-          <Text className='header-text' type="header-secondary">{t('webex.calling')}</Text>
-          <ButtonCircle className='cancel-button' ghost onPress={onCancel} aria-label={t("voiceover.addSpeedDial.closeBtn")} onKeyDown={handleEnterKeyPress}>
+          <Text className='header-text' type="header-secondary" aria-hidden="true">{t('webex.calling')}</Text>
+          <ButtonCircle className='cancel-button' ghost onPress={onCancel} aria-hidden="true" aria-label={t("voiceover.addSpeedDial.closeBtn")} onKeyDown={handleEnterKeyPress}>
```

Comment:
Could you please explain why we added aria-hidden="true" to the cancel button?

Reply:
When using the modal dialog, if we do not use aria-hidden="true", it will announce all the elements of the modal dialog, which is not necessary. Also it depends on browser.